spec/helpers/thirdParty.js

```diff
@@ -77,22 +77,6 @@ describe('third party handlebars-helpers', function() {
 
     });
 
-    describe('date helpers', function() {
-
-        describe('contains moment', function() {
-            it('renders the date in the format specified', function(done) {
-                const now = new Date();
-                runTestCases([
-                    {
-                        input: `{{#moment "1 year ago" "YYYY"}}{{/moment}}`,
```

Comment:
Are we removing this functionality? If not, can you replicate this test case?

Reply:
removing